framework/SwiftPlot/BarChart.swift

```diff
@@ -0,0 +1,297 @@
+import Foundation
```

Comment:
the indentation is showing up pretty weirdly for me in this file (and some of the others in this PR). there are tabs. I think if you change them to 4-space indent it'll fix it.

Reply:
I'll correct this using the formatter